src/Stream.php

```diff
@@ -359,6 +361,14 @@ private function isValidStreamResourceType(mixed $resource): bool
         }
 
         if ($resource instanceof GdImage) {
```

Comment:
I get type errors for pretty much all methods in Stream when GdImage is used as resource.

```
Uncaught TypeError: stream_get_contents(): Argument #1 ($stream) must be of type resource, GdImage given
Uncaught TypeError: stream_get_meta_data(): Argument #1 ($stream) must be of type resource, GdImage given
Uncaught TypeError: fseek(): Argument #1 ($stream) must be of type resource, GdImage given
```
and so on

Reply:
It did not work at all for a long time. I suggest "support" is dropped without deprecation as no actual users will be affected.